### PR TITLE
added support for converting IDs and codenames to cards/names

### PR DESCRIPTION
**Pull request by ethn**

### CODE REVIEW

File: mod/wikirate/spec/config/initializers/core_extensions_spec.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
+describe CoreExtensions do
+  context String do
+    it "tests whether a string represents a number" do
+      expect("6".number?).to eq(true)
+      expect("Yomama".number?).to eq(false)
+    end
+  end
+
+  context CoreExtensions::PersistentIdentifier do
+    it "converts into a cardname" do
```
Comment (xithan) [inline mod/wikirate/spec/config/initializers/core_extensions_spec.rb:10]: subject doesn't seem right here. I miss something like `describe "#name"` but at least there are tests.

**Comment by xithan** [comment]
why do we have this in wagn and wikirate?

File: mod/wikirate/spec/config/initializers/core_extensions_spec.rb (hunk 1 above)
Comment (ethn) [reply mod/wikirate/spec/config/initializers/core_extensions_spec.rb:10]: good point.  will fix.

**Comment by ethn** [comment]
I started in WikiRate and then realized that was dumb.  it's out of wikirate now in the research group branch (though I forgot to move the tests.

**Pull request merged**